scala3doc/src/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala

```diff
@@ -17,13 +17,13 @@ class ImplicitMembersExtensionTransformer(ctx: DokkaContext) extends Documentabl
 
     def expandMember(outerMembers: Seq[Member])(c: Member): Member =
       val companion = c match
-        case classlike: DClass => ClasslikeExtension.getFrom(classlike).flatMap(_.companion).flatMap(classlikeMap.get)
+        case classlike: DClass => ClasslikeExtension.getFrom(classlike).flatMap(_.companion).map(classlikeMap)
```

Comment:
This change may not be safe (it will throw exception when `classlikeMap` does not contains companion

Reply:
I changed it to safe call (`Map.get`) and log when classlikeMap is missing companion